Dockerfile.assisted-service-build

```diff
@@ -1,12 +1,14 @@
-FROM registry.ci.openshift.org/openshift/release:golang-1.16
-
+FROM quay.io/centos/centos:stream8
 ENV GO111MODULE=on
 ENV GOFLAGS=""
+ENV VIRTUAL_ENV=/opt/venv
+ENV GOPATH=/go
+ENV PATH="$VIRTUAL_ENV/bin:$GOPATH/bin:$PATH"
 
 COPY ./hack/setup_env.sh ./dev-requirements.txt ./
+RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && dnf -y update && dnf -y install make git jq golang python3 && dnf clean all
```

Comment:
Is that necessary to install the tools in here? I see it is installed as part of `setup_env`